src/Makefile.inc

```diff
@@ -0,0 +1,340 @@
+
+# We want to turn on INCLUDE_FS by default
+INCLUDE_FS ?= 1
+
+# Use VERBOSE=1 to enable verbose make
+PROGMODE ?= STATUS_SCREEN
+
```

Comment:
What is this?  I can't find any reference to it anywhere else.  No idea why we have it.

Reply:
i don't know.